docs/cubeviz/import_data.rst

```diff
@@ -12,29 +12,33 @@ be defined in ``specutils`` instead of being duplicated in Jdaviz.
 and hence should address most astronomical spectrum use cases.
 
 Cubeviz will automatically parse the data into the multiple viewers as described in
-:ref:`cubeviz-display-cubes`. Data loaded into Cubeviz MUST contain valid WCS
+:ref:`cubeviz-display-cubes`. For the best experience, data loaded into Cubeviz should contain valid WCS
```

Comment:
does this require #1734 first?

Reply:
Well... it is already possible, just there is a bug and the collapsed spectrum won't show properly. 🤷 

I wasn't sure where to draw the line when trying to break up the original PR into smaller pieces.

Reply:
ok, as long as we do support data without WCS before 1734 to some extent, I think this is fine.  Otherwise, I would just vote to get 1734 in first before this.